src/lib/theme.ts

```diff
@@ -0,0 +1,23 @@
+'use server';
+
+import { cookies } from 'next/headers';
+
+const THEME_COOKIE_NAME = 'relisten_theme';
+
+export type Theme = 'light' | 'dark';
+
+export async function getTheme(): Promise<Theme | null> {
```

Comment:
this will create a server action (since use server) which isn't necessary since its only called from the server.

setTheme can be a server action, but it should include validation (theme can only be light|dark)